### PR TITLE
Add OAB documentation

### PR DESCRIPTION
**Pull request by BearSteve**

### CODE REVIEW

**Comment by BearSteve** [comment]
Eagle eyes @tomstumshais as always. Thanks!

File: source/products/oab/household/webapp.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
+Webapp
+======
+
+Example Kasko JS embed code (OAB - OAB - Hausrat)
```
Comment (Marcisbee) [inline source/products/oab/household/webapp.rst:4]: Why `OAB` x2 here?

Comment (tomstumshais) [reply source/products/oab/household/webapp.rst:4]: Double or nothing! ✌️

Comment (BearSteve) [reply source/products/oab/household/webapp.rst:4]: I just copied it from dashboard 🤔 But yeah, I think writing it once makes it clear either way.

Comment (BearSteve) [reply source/products/oab/household/webapp.rst:4]: So there are 2 flows, OAB and Schwarzwalder, that's why 🤷‍♂️

**Pull request merged**